src/Backends/Cubics/CubicBackend.h

```diff
@@ -31,6 +31,7 @@ class CubicResidualHelmholtz;
 class AbstractCubicBackend : public HelmholtzEOSMixtureBackend  {
 protected:
     shared_ptr<AbstractCubic> cubic;
+    std::vector<CubicLibrary::CubicsValues> components; ///< The components that are in use
```

Comment:
In `HelmholtzEOSMixtureBackend` components is of type `std::vector<CoolPropFluid>`.
The two types are not compatible, for example, in `CoolPropFluid` molar mass is called by `components[i].molar_mass()`, whereas in the cubic it is `components[i].molemass`.
This also explain the need to a new cubic `calc_molar_mass()` definition.